.github/workflows/build.yml

```diff
@@ -60,6 +63,28 @@ jobs:
       if: runner.os == 'macOS'
       run: |
         brew install libuv swig pkgconfig jsoncpp
+    - uses: actions/checkout@v2
```

Comment:
This looks redundant, duplicating the original proton checkout, with cleanup side effects.